dll/win32/browseui/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ add_library(browseui MODULE
 
 set_module_type(browseui win32dll UNICODE)
 target_link_libraries(browseui shellbars shellfind uuid wine cpprt atl_classes)
-add_importlibs(browseui uxtheme shlwapi shell32 comctl32 gdi32 ole32 oleaut32 user32 advapi32 mpr msvcrt kernel32 ntdll)
+add_importlibs(browseui uxtheme shdocvw shlwapi shell32 comctl32 gdi32 ole32 oleaut32 user32 advapi32 mpr msvcrt kernel32 ntdll)
```

Comment:
Putting the comment here, but just to say: Win2003 browseui indeed imports shdocvw for its functionality, and indeed, it uses this AddUrlToFavorites function, so for me @whindsaks suggestion and what you've done in the latest revision of this PR is OK:
![image](https://github.com/user-attachments/assets/fee4ef26-06f3-461a-855a-c2117218c6be)